src/containers/TemplateBuilder/template/helpers.ts

```diff
@@ -0,0 +1,32 @@
+import { customAlphabet } from 'nanoid'
+import { VersionObject } from '../useGetTemplates'
+
+// Use nanoid to generate unique template version IDs
+export const getTemplateVersionId = customAlphabet('abcdefghijklmnopqrstuvwxyz1234567890', 6)
```

Comment:
I went with a simple 6-char alphanumeric value, since we'll often be comparing versionIds by eye, or verbally. And something like `ba39d9` is a heck of a lot easier to quickly parse mentally than `V1StGXR8_Z5jdHi6B-myT`!

There's still over 2 billion possibilities though, and so the probability of a collision (for a single template, remember) is [pretty miniscule](https://zelark.github.io/nano-id-cc/)